src/code/z_camera_data.c

```diff
@@ -59,8 +59,15 @@ char sCameraModeNames[][12] = {
     "HANG       ", "HANGZ      ", "FREEFALL   ", "CHARGE     ", "STILL      ", "PUSHPULL   ", "BOOKEEPON  ",
 };
 
+/*=====================================================================
+ *                   GENERAL NORMAL0 CAMERA DATA
```

Comment:
maybe make it more obvious `NORMAL0` is a setting name? reuse the enum like `CAM_SET_NORMAL0` ? or `GENERAL NORMAL0 SETTING CAMERA DATA` (don't really like it)

also are you sure about all caps?